kotlin-android/app/src/main/java/com/mongodb/tasktracker/ProjectActivity.kt

```diff
@@ -0,0 +1,114 @@
+package com.mongodb.tasktracker
+
+import android.app.AlertDialog
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.EditText
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+import com.mongodb.tasktracker.model.Project
+import com.mongodb.tasktracker.model.ProjectAdapter
+import com.mongodb.tasktracker.model.User
+import io.realm.Realm
+import io.realm.RealmList
+import io.realm.kotlin.where
+import io.realm.mongodb.sync.SyncConfiguration
+import java.util.*
+
+class ProjectActivity : AppCompatActivity() {
```

Comment:
nit: Could we add a comment that gives a quick overview of what this Activity does?